lib/stacks/infra/core.ts

```diff
@@ -454,7 +457,7 @@ export class CoreStack extends Stack {
             role: props.mlSpaceAppRole
         });
 
-        new AwsCustomResource(this, 'populate-allowed-instance-types', {
+        new AwsCustomResource(this, 'initial-app-config-deployment-001', {
```

Comment:
nit: do we need the `001`?

Reply:
This only ever runs once. So for us to get it to re-run we have to modify the name. Instead of coming up with unique names each time I figured a counter like this made more sense.